javascript/new-src/src/constants.js

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
Is this a personal choice to change them? It's upset the licence enforcer. So could we change it back, or make all the old SRC the same too and change this (although that may upset the other linter):
https://github.com/OpenLightingProject/ola/blob/master/scripts/enforce_licence.py#L142